### PR TITLE
Devfile  screenshots

### LINKED ISSUE #4886: [Screenshots] Devfile under 'How it Works See It in Action' section
### Current Behavior
Currently, [every integration](https://layer5.io/cloud-native-management/meshery/integrations/devfile) has only two old screenshots of the designer and visualizer in the 'How it Works See It in Action' section.

### Desired Behavior
- Add new images and videos from the latest version of the software ([MeshMap](https://layer5.io/cloud-native-management/meshmap))
- Add screenshots for [Devfile](https://layer5.io/cloud-native-management/meshery/integrations/devfile) integration.

### Acceptance Tests
- Ensure your screenshot represents the use of a few specific integrations.
- The screenshot from the Designer mode should contain a valid design, refer to [meshery catalog](https://meshery.io/catalog) to use and view published designs.
- Add both Dark Mode and Light Mode screenshots.
- The screenshot should not be a snippet, take the full screenshot of your browser.
- Use Incognito mode on Chrome browser, to ensure your bookmark bar or any other items are not shown in the screenshot/video.

---

<h2>Contributor Resources and <a href="https://layer5.io/community/handbook">Handbook</a></h2>

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- 📚 See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md).
-  🎨 Wireframes and [designs for Layer5 site](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](https://slack.layer5.io).

Join the Layer5 Community by submitting your [community member form](https://layer5.io/newcomer).

### PR DESCRIPTION
**Pull request by narasimha-1511**
**Description**
Please Review my PR
This PR fixes #4886 

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 8ae83b8b4f7593d70aab2004c183f6f728df05d5 at: https://65c4fdf93838cf2396dd18bc--layer5.netlify.app

**Comment by saurabh100ni** [comment]
Hi @narasimha-1511  ,
Thanks for your PR! I encourage you to share this in the Website Meeting tomorrow at 6:30 PM IST. Share your screen, and walk us through it. It's cool to take and share feedback. Check the [Doc](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit) here to include this PR.

You're welcome at the meeting

Check to [community calendar](https://layer5.io/community/calendar) for more details of meetings at Layers.

**Comment by saurabh100ni** [comment]
@narasimha-1511, I guess you already know what needs to be changed here 😉

**Comment by narasimha-1511** [comment]
changes done @saurabh100ni 😄

**Comment by l5io** [comment]
🚀 Preview for commit f3b9cfe7b147ed8a3f06c24555e52f68d11e7136 at: https://65cc7c304419d787fb0f8004--layer5.netlify.app

**Review by saurabh100ni — APPROVED** [review]

**Comment by saurabh100ni** [comment]
@narasimha-1511, do you know what we do with that checkbox in PR description 👀?

**Comment by narasimha-1511** [comment]
I have Signed my commits 😅 .

**Comment by narasimha-1511** [comment]
hey @vishalvivekm this PR LGTM can you merge it?

**Comment by narasimha-1511** [comment]
this PR LGTM @iArchitSharma

File: src/collections/integrations/devfile/index.mdx [hunk 1]
```diff
@@ -15,8 +15,8 @@ featureList: [
   "Central location management so updates can be applied once and be properly aligned across development teams."
 ]
 workingSlides: [
-  ../_images/meshmap-visualizer.png,
-  ../_images/meshmap-designer.png
+  ../_images/meshmap-designer.png,
```
Comment (iArchitSharma) [inline src/collections/integrations/devfile/index.mdx:18]: @narasimha-1511  aren't the images supposed to be inside `_images/devfile` directory

**Review by iArchitSharma — CHANGES_REQUESTED** [review]

File: src/collections/integrations/devfile/index.mdx (hunk 1 above)
Comment (narasimha-1511) [reply src/collections/integrations/devfile/index.mdx:18]: they are inside devfile folder these images are automatically used in the integration from the folder you can see it here:https://65cc7c304419d787fb0f8004--layer5.netlify.app/cloud-native-management/meshery/integrations/devfile

**Comment by leecalcote** [comment]
Thank you.

**Pull request merged**